### ISSUE TITLE
Deluge - Labels not working

### ISSUE THREAD

**Issue by Lyechee** [tympanix/Electorrent#174]
**Client:**
- [ ] µTorrent
- [ ] qBittorrent
- [ ] Transmission
- [ ] rTorrent
- [*] Deluge

**Client Version:**
1.3.15
**Operating System:**
w10
**Application Version:**
2.7.1
**Description:**
Can't see the labels of the torrents loaded in Electorrent. Can't also give a label to a torrent, nor create a new label.

**Comment by seanfarley** [comment]
Can confirm that deluge 2.0 labels aren't loading / working.

**Comment by tympanix** [comment]
Yes, Deluge v2 is not officially supported yet. As far as i know, the v1 implementation works partially.
